core/src/account_provider.rs

```diff
@@ -247,6 +247,10 @@ impl<'a> ScopedAccount<'a> {
             phantom: PhantomData::default(),
         }
     }
+
+    pub fn into_owned(self) -> DecryptedAccount {
```

Comment:
Why not ToOwned?
https://doc.rust-lang.org/std/borrow/trait.ToOwned.html

Reply:
To implement `ToOwned` for `ScopedAccount`, `Borrow` should be implemented for `DecryptedAccount`. The relationship is bidirectional, but package dependency isn't.
And also, I introduced `ScopedAccount` because I wanted developers to think twice before assigning `DecryptedAccount` somewhere permanently. But to implement `ToOwned`, `derive(Clone)` for `DecryptedAccount` is needed, and `ToOwned` has `&self` which makes the ownership of `DecryptedAccount` is blurry.

Reply:
I think you should name it differently if you want users to think twice before assigning it permanently.
How about `into_decrypted`?

Reply:
I thought `disclose` would be good.

Reply:
Looks good.

Reply:
It is resolved in https://github.com/CodeChain-io/codechain/pull/1278